presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixMetadata.java

```diff
@@ -351,7 +351,7 @@ private JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorTab
                     typeStatement += " not null";
                     pkNames.add(columnName);
                 }
-                columnList.add(format("%s %s", columnName, typeStatement));
+                columnList.add(format("%s %s", getEscapedArgument(columnName), typeStatement));
```

Comment:
@vincentpoon should we go thru `toMetadataCasing` here before calling `getEscapedArgument`?

Reply:
Hmm this code doesn't look right to me (I think it was from another PR I based my original one off of).  Firstly, `connection.getMetaData().storesUpperCaseIdentifiers()` would always return true, so the method `toMetadataCasing` isn't really needed.

Phoenix uppercases identifiers if they aren't quoted.
If they are quoted, then it's case sensitive.

Is there a way for us to tell in Presto whether the user quoted the identifier or not?  Because it seems we should base the uppercase decision off that.

Reply:
> Is there a way for us to tell in Presto whether the user quoted the identifier or not? Because it seems we should base the uppercase decision off that.

not yet,  working on that -- https://github.com/prestosql/presto/issues/17